relations/src/preimage/relation.rs

```diff
@@ -0,0 +1,100 @@
+use ark_ff::BigInteger256;
+// This relation showcases how to use Poseidon in r1cs circuits
```

Comment:
nit: let's move it either to very top of this module or to `mod.rs`

Reply:
done